src/api/app/views/layouts/webui/_notification_toolbar.html.haml

```diff
@@ -0,0 +1,11 @@
+- if notification
+  .notification-toolbar.sticky-top.navbar.navbar-light.bg-light.border-bottom.border-gray-400
+    - title, icon = notification.unread? ? ['Mark as read', 'fa-check'] : ['Mark as unread', 'fa-undo']
```

Comment:
And maybe move this logic into a helper

Reply:
Moved to helper. Not sure about the name for the function, I've chosen `title_and_icon_for_read_button`. If you have better name in mind, please let me know.

Reply:
You can move the `link_to` also and the maybe you can rename the helper to `link_for_notification` or something like this

Reply:
Done.